src/test/testing-library/NotificationError.test.tsx

```
@@ -0,0 +1,211 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { t } from 'i18next';
+
+import NotificationError from '../../features/landing/components/Notification/NotificationError';
+import styles from '../../features/landing/components/Notification/styles';
+
+import { backToFormButtonText, buttonRole, retrySubmitButtonText } from './constants';
+
+const errorImgAltText: string = t('notifications.error.images.error');
+export const errorTitle: string = t('notifications.error.title');
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)